Doc/Tutorial/chapter_testing.tex

```diff
@@ -332,11 +332,7 @@ \section{Writing doctests}
 
 # Exclude modules with online activity
 # They are not excluded by default, use --offline to exclude them
-ONLINE_DOCTEST_MODULES = [
-    "Bio.Entrez",
-    "Bio.ExPASy",
-    "Bio.TogoWS",
-]
```

Comment:
How did you do this LaTeX file? I thought da18e831bf523768ee4265e0ba2cacc264dc34f8 using ``blacken-docs`` would have covered this.

Reply:
I was a bit confused by that one. In the other #2827 it was done automatically. When I ran `blacken-docs -t py38 chapter_testing.tex` on this file alone, nothing happened. I guess it might have been one of the other tools I tried in between?

Anyway, blacken-docs did check the file. It seems as both versions are acceptable for black. Maybe pyupgrade did it in the first PR?

Reply:
Curious - the new version is shorter, so let's leave it.